src/cider/nrepl.clj

```diff
@@ -436,7 +436,7 @@ Depending on the type of the return value of the evaluation this middleware may
     :requires #{#'session #'wrap-print}
     :expects #{}
     :handles {"stacktrace" {:doc "Return messages describing each cause and stack frame of the most recent exception."
-                            :optional wrap-print-optional-arguments
+                            :optional (assoc wrap-print-optional-arguments "stacktrace" "The stacktrace to be parsed and analyzed as a string.")
```

Comment:
@bbatsov @vemv I am wondering if I should a new "parse stacktrace" op instead off adding this new optional parameter? It feels cleaner instead of hijacking the existing op to me. Wdyt?

Reply:
I agree. Generally we've been using action-like names for most ops (e.g. `eval`, `clone`, etc), so the stacktrace op was named somewhat poorly in that regard anyways, as it was super clear what does it do by its name alone.